stable/concourse/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: concourse
-version: 4.0.0
+version: 4.1.0
```

Comment:
Given that we're removing something that could be previously set (and people could be relying on that), I'd suggest making this a major bump

Reply:
My Rationale was that it is not that big of a change, but I still agree that we can bump Major.